frontend/src/data/constants.ts

```diff
@@ -275,3 +276,10 @@ export const COUNTRY_CODES_TO_NAME = {
   ZM: 'Zambia',
   ZW: 'Zimbabwe',
 }
+
+export const COUNTRY_CODE_OPTIONS = enumKeys(COUNTRY_CODES_TO_NAME).map(
+  (key) => ({
+    label: COUNTRY_CODES_TO_NAME[key],
+    value: key,
+  }),
```

Comment:
Suggestion (non-blocking): alternatively could do

```typescript
export const COUNTRY_CODE_OPTIONS = Object.entries(COUNTRY_CODES_TO_NAME).map(
  ([countryCode, countryName]) => ({
    label: countryName,
    value: countryCode
  })
)
```

Reply:
Ooooh I forgot about that method 🙏